assets/js/notifications.js

```diff
@@ -0,0 +1,4 @@
+setTimeout(() => {
+  const notifications = document.querySelectorAll(".notification");
```

Comment:
Strings must use singlequote.

[Read more about it here.](http://eslint.org/docs/rules/quotes)